Resources/Prototypes/Nuclear14/Entities/Objects/Weapons/Guns/Snipers/snipers.yml

```diff
@@ -25,3 +34,38 @@
       - CartridgeAntiMaterial
     capacity: 5
     proto: CartridgeAntiMaterial
+
+- type: entity
+  name: lever action carbine
+  parent: BaseWeaponSniper
+  id: N14WeaponLeverCarbine
+  description: A lever action short carbine. Uses .30 rifle ammo.
+  components:
+  - type: Sprite
+    sprite: Nuclear14/Objects/Weapons/Guns/Snipers/levercarbine.rsi
+  - type: Item
+    sprite: Nuclear14/Objects/Weapons/Guns/Snipers/levercarbine.rsi
+  - type: Gun
+    fireRate: 3
+    capacity: 14
```

Comment:
This needs to go on the `  - type: BallisticAmmoProvider` component for tests to pass.